File: java/program69.java
import java.util.*;

class program69
{
    public static int CalculatePower(int iBase,int iPower)
    {
        int result =   1;

        for(int i = 1; i <= iPower; i++)
        {
            result = result * iBase;
        }
        return result;
    }

    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Base : ");
        int value1 = sobj.nextInt();

        System.out.println("Enter power : ");
        int value2 = sobj.nextInt();

        int ret = CalculatePower(value1,value2);
        System.out.println("Result is : "+ret);
    }
}

File: cpp/program69.cpp
// program46.c

#include<iostream>
using namespace std;

class Number
{
    public:
        int iBase;
        int iPower;

        Number(int X,int Y)
        {
            iBase = X;
            iPower = Y;
        }

        int CalculatePower()
        {
            int iCnt = 0;
            int iResult = 1;

            for(iCnt = 1; iCnt <= iPower; iCnt++)
            {
                iResult = iResult * iBase;
            }
            return iResult;
        }
};

int main()
{
    int iValue = 0;
    int iValue1 = 0;

    cout<<"Enter Base \n";
    cin>>iValue;

    cout<<"Enter power \n";
    cin>>iValue1;

    Number obj(iValue,iValue1);
    
    cout<<"Result is : "<<obj.CalculatePower()<<"\n";

    return 0;
}